pkg/sources/github/github.go

```diff
@@ -594,17 +601,14 @@ func (s *Source) enumerateWithToken(ctx context.Context, apiEndpoint, token stri
 
 		for _, org := range s.orgsCache.Keys() {
 			logger := s.log.WithValues("org", org)
-			if err := s.getReposByOrg(ctx, org); err != nil {
-				logger.Error(err, "error fetching repos by org")
-			}
-
-			if err := s.getReposByUser(ctx, ghUser.GetLogin()); err != nil {
-				logger.Error(err, "error fetching repos by user")
```

Comment:
This appears to be a bug. It calls `s.getReposByUser` for the authenticated user on every iteration.